docs/notebooks/neural_dual.ipynb

```diff
@@ -12,7 +12,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    from functools import partial
</pre><p>Use <code>import functools</code> and <code>import dataclasses </code>rather than <code>from ...</code> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>